gufe/components/explicitmoleculecomponent.py

```diff
@@ -31,6 +32,16 @@ def _ensure_ofe_name(mol: RDKitMol, name: str) -> str:
     elif name == "":
         name = rdkit_name
 
+    # sanitize the name before we set it
+    # list of characters to replace
+    to_replace = [" ", "/"]
+    if any([i in name for i in to_replace]):
+        # strip leading and trailing whitespace
+        name = name.strip()
+        for i in to_replace:
+            name = name.replace(i, "-")
```

Comment:
I'm getting some weird deja-vu of a discussion had like 2 years ago with Jenke? or Swenson? Something along the lines of "this could break SMILES-based names". Thoughts on this? I'm tempted at this point to tell folks to stop using SMILES as molecule names...

Reply:
Ahh good point this will break some smiles-based names, yes we should advise people to use names suitable to identify the ligand and to be used as an output file in downstream workflows?

Reply:
So one question here is if the problem with the names is at the gufe level or at the openfe level. i.e. is this just a problem because of our CLI and we're enforcing behaviour on everyone?

Reply:
One silly idea here is to have a sanitize option around the class variable and sanitize on the fly :P